src/modules/governance/proposal/VotersListItem.tsx

```diff
@@ -0,0 +1,81 @@
+import { ExternalLinkIcon } from '@heroicons/react/solid';
+import { Avatar, Box, SvgIcon, Typography } from '@mui/material';
+import makeBlockie from 'ethereum-blockies-base64';
+import { useState } from 'react';
+import { FormattedNumber } from 'src/components/primitives/FormattedNumber';
+import { Link } from 'src/components/primitives/Link';
+
+import { textCenterEllipsis } from '../../../helpers/text-center-ellipsis';
+import type { GovernanceVoter } from './VotersListContainer';
+
+type VotersListItemProps = {
+  voter: GovernanceVoter;
+};
+
+export const VotersListItem = ({ voter }: VotersListItemProps): JSX.Element | null => {
+  const { address, ensAvatar, ensName, twitterAvatar } = voter;
+  const [useBlockie, setUseBlockie] = useState(false);
+
+  // If voter has an ENS name, show it, otherwise, show address. Both will be abbreviated, except short ENS names.
+  const displayName = ensName
+    ? ensName.length > 18
+      ? textCenterEllipsis(ensName, 12, 3)
+      : ensName
+    : textCenterEllipsis(address, 8, 3);
+
+  // For avatars, the order of precedence is Twitter, then ENS, then default Blockie
+  // If ENS avatar does not exist, then use Blockie
+  const displayAvatar = useBlockie
+    ? makeBlockie(address)
+    : twitterAvatar ?? ensAvatar ?? makeBlockie(address);
+
+  // Don't show any results that come back with zero or negative voting power
+  if (voter.votingPower <= 0) return null;
```

Comment:
I think here we need to look at the `votingPower` on the proposal that they voted on. I think this is the users current voting power.

Reply:
Oh weird, yeah definitely worth taking another look once we get all synced. I was thinking we'd need to look at the `votingHistory` prop for this proposal to get what power was when they voted.

I _think_ `voter.votingPower` is their current power, so it could be 0 if they don't have any aave

Reply:
Ahh that makes sense and could definitely be what was going on here.  Will take a look once resynced and inspect the log statement that contains `voters`.  Could be an easy fix to just `voter.votingHistory.filter(h => h.proposal.id === proposal.id)` at the container level during the transform statements, and so that it may easily be passed into `VoterListItem` and easily consumed.  We'd also alter the type as well.

On that note, yes I think it may make sense to create a type for the server data, and that's exactly what I was doing here.  It needs to be fleshed out fully before merging this PR into `main`, IMO

Reply:
This doesn't quite match up completely.  The voting power on each voting history item is vastly larger than the current voting power.  These are determined in the backend. Do we need to factor in voting weight to reach the correct number?  See recent changes that pull this number and display it: https://github.com/aave/interface/pull/1244/commits/bd5ac66361b1d439d952b8fde91780ef04ffc417